pkg/acctest/test.go

```diff
@@ -62,15 +63,15 @@ func PreCheck(t *testing.T) {
 			t.Fatalf("Failed to set custom base URL: %v", err)
 		}
 
-		configErr := Provider.Configure(context.Background(), terraform.NewResourceConfigRaw(nil))
+		configErr := Provider.Configure(context.Background(), (*terraform2.ResourceConfig)(terraform.NewResourceConfigRaw(nil)))
 		if configErr != nil {
 			t.Fatalf("Failed to configure provider %v", configErr)
 		}
 	})
 }
 
 func GetArtifactoryUrl(t *testing.T) string {
-	return test.GetEnvVarWithFallback(t, "ARTIFACTORY_URL", "JFROG_URL")
+	return testutil.GetEnvVarWithFallback(t, "ARTIFACTORY_URL", "JFROG_URL")
```

Comment:
We should swap the ordering of these 2 env vars as well

Reply:
yes.